AddOns/BasicFinancials/app/src/codeunit/BasicFinancialsMgmt.Codeunit.al

```diff
@@ -0,0 +1,125 @@
+codeunit 20601 "Basic Financials Mgmt BF"
+{
+    Access = Internal;
+
+    var
+        AzureADLicensing: codeunit "Azure AD Licensing";
+        NotSupportedLicenseErr: Label 'To deploy the Basic Financials extension the Basic Financials license must be assigned to at least one user.';
+        NotSupportedLocalesErr: Label 'The Basic Financials Extension can only available in Denmark and Iceland.';
+        NotSupportedUserErr: Label 'The user who deploys the Basic Financials extension must have the Super User permission set.';
+        NotSupportedCompanyErr: Label 'The Basic Financials Extension, can only be deployed, when exactly one Company is present in the Environment.';
```

Comment:
The Basic Financials extension can only be deployed when exactly one company exists in the tenant.

Reply:
I don't think that's right. The code is testing for exactly one company exists in the environment. The code is not testing every Environment in the tenant.